brambling/management/commands/send_daily_emails.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import traceback
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from brambling.mail import DailyDigestMailer
+from brambling.models import Person
+
+
+class Command(BaseCommand):
+    def get_recipients(self):
+        return Person.objects.filter(
+            notify_new_purchases=Person.NOTIFY_DAILY,
```

Comment:
Sorry if my comment was a little vague. I think what I'm saying is that there is nothing inherently daily about the send daily emails command. Right now it looks like (and correct me if I'm wrong) that the command will send emails about transactions that have happened since the last time the command was run, and cron is the thing that ensures it happens only once per day.

Whereas if there were a filter here to only select, e.g., people with `last_new_purchases_digest_sent` older than 24 hours, there's no chance of anyone being emailed their daily digest twice in that interval.